src/sagemaker/experiments/run.py

```diff
@@ -29,6 +29,7 @@
 from sagemaker.experiments._helper import (
     _ArtifactUploader,
     _LineageArtifactTracker,
+    _DEFAULT_ARTIFACT_PREFIX,
```

Comment:
nit the name of this sounds like it will prefix the actual name of the artifact but its really a prefix for a s3 path where the associated files will be stored

Reply:
Yes, it's a bit confusing to use this name here. But `_DEFAULT_ARTIFACT_PREFIX` is not introduced in this PR, it's already defined in [_helper.py](https://github.com/aws/sagemaker-python-sdk/blob/9caf00f0d266a1bac9275d67fada28b5c849310a/src/sagemaker/experiments/_helper.py#L28). There is a variable called `artifact_prefix`, so I'm assuming that is why we called the default variable to be `_DEFAULT_ARTIFACT_PREFIX`.

A workaround could be change the following method defined in `_ArtifactUploader`.
```
class _ArtifactUploader(object):
    """Artifact uploader"""
    def __init__(
        self,
        trial_component_name,
        sagemaker_session,
        artifact_bucket=None,
        artifact_prefix=None, # HERE!!!
    ):
        self.sagemaker_session = sagemaker_session
        self.trial_component_name = trial_component_name
        self.artifact_bucket = artifact_bucket
        self.artifact_prefix = (
            _DEFAULT_ARTIFACT_PREFIX if artifact_prefix is None else artifact_prefix # HERE!!!
        )
        self._s3_client = self.sagemaker_session.boto_session.client("s3")
```
In this way, we don't need to import `_DEFAULT_ARTIFACT_PREFIX` to run.py